build/buildx/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Buildx
+title: Working with Buildx
 description: Working with Docker Buildx
-keywords: build, buildx, buildkit
+keywords: build, buildx, BuildKit
```

Comment:
Good question; I _think_ either way _may_ work, as long as we're consistently using the same.

Looks like the search auto-complete uses them case-insensitive, but it may be that they're used different elsewhere;

<img width="612" alt="Screenshot 2022-07-26 at 11 04 03" src="https://user-images.githubusercontent.com/1804568/180968442-ee7ad90b-94d3-476d-bd7b-e69171325379.png">

"elsewhere" could be as part of an URL and/or anchor, in which case it's _possible_ that they're treated as "different tags". I guess "lowercase" would be the safest option for these 🤔

Reply:
Ah indeed there is the auto-complete search but I was thinking about search engines and the `meta name="keywords"` that is case-insensitive. See also https://github.com/docker/docker.github.io/pull/14642#discussion_r878979334 about `keywords` meta tag abuse. So might only be relevant for our auto-complete.

Reply:
So, in your [previous example](https://github.com/docker/docker.github.io/pull/14642#discussion_r878979334) all of them are like Docker, BuildKit, etc.
![image](https://user-images.githubusercontent.com/103652162/180986825-54821f69-b529-43df-ba2d-9bc7cf75c3b2.png)

Comment: I agree that making them all lowercase allows less room for creating multiple like (docker, Docker, dockerfile, Dockerfile) in terms of metadata, in terms of the presentation (they are visible) they make more sense with case. It would be great to decouple that. As in, write them in suitable case (rendering) and them being always interpretated in case insensitive way(metadata). 

Question for @usha-mandya do we then change all keywords to be lowercase in our docs sets?

Reply:
@dockertopia Agree on having a standard for keywords. Suggest we use lowercase going forward. 

We can create a ticket in the backlog to retrospectively update all keywords to be lowercase. It's not a high priority as the search results are case-insensitive.

Reply:
Thanks @usha-mandya. Sounds good.